### PR TITLE
SAP IDoc behavior

### PR DESCRIPTION
**Pull request by zweedao**

### CODE REVIEW

**Comment by zweedao** [comment]
@graceliching can you help review

**Comment by zweedao** [comment]
@graceliching 👌

File: connectors/sap/trigger-new-idoc.md [hunk 1]
```diff
@@ -7,7 +7,14 @@ date: 2018-12-06 06:00:00 Z
...
 ![Trigger New IDoc](/assets/images/connectors/sap/trigger-new-idoc.png)
 
-This trigger allows you to listen to and receive IDocs from an on-premise SAP system, or from an SAP system on a server behind the corporate firewall. Note that you must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this trigger.
+This trigger allows you to listen to and receive IDocs from an on-premise SAP system, or from an SAP system on a server behind the corporate firewall.
+
+A few things to take note:
+
+- You must [configure IDocs in your SAP system](https://docs.workato.com/connectors/sap.html#configure-idoc-in-sap-to-work-with-workato) before using this trigger.
+
+- Currently, 1 IDoc can only be listened by 1 recipe. Using multiple recipes to listen to the same IDoc will result in lost IDoc. We will make enhancement to improve this in the future, but for now a simple workaround is to use [Callable recipes](/features/callable-recipes.md). For details on how to do this, please see the [Troubleshooting section](#Troubleshooting) at the end of this doc.
```
Comment (graceliching) [inline connectors/sap/trigger-new-idoc.md]: Currently, **an IDoc can only be used in the trigger of 1 recipe.** Using the same IDoc in triggers of more than 1  recipes will result in some triggers not picking up changes to that IDoc. Workato is making enhancements to improve this in the near future, but at present, a simple workaround is to use [Callable recipes](/features/callable-recipes.md). For details on how to do this, please see the [Troubleshooting section](#Troubleshooting) at the end of this doc.

File: connectors/sap/trigger-new-idoc.md [hunk 2]
```diff
@@ -19,3 +26,23 @@ This trigger allows you to listen to and receive IDocs from an on-premise SAP sy
 The trigger will output all segments and data fields of the IDoc received from SAP. In this sample IDoc, `EDI_DC40` and `E1MARAM` segments.
 
 ![Trigger New IDoc output](/assets/images/connectors/sap/trigger-new-idoc-output.png)
+
+## Troubleshooting
+
+### I use multiple recipes to listen to the same IDoc and some recipes miss the IDoc
+
+Currently 1 IDoc can only be listened by 1 recipe. Using multiple recipes to listen to the same IDoc will result in lost IDoc.
```
Comment (graceliching) [inline connectors/sap/trigger-new-idoc.md]: Currently, **an IDoc can only be used in the trigger of 1 recipe.** Using the same IDoc in triggers of more than 1 recipes will result in some triggers not picking up changes to that IDoc.

File: connectors/sap/trigger-new-idoc.md [hunk 3]
```diff
@@ -19,3 +26,23 @@ This trigger allows you to listen to and receive IDocs from an on-premise SAP sy
...
+
+A simple workaround is to use [Callable recipes](/features/callable-recipes.md). You will create 1 recipe to listen to the IDOC, then call other recipes to execute different workflows. Here is an example:
```
Comment (graceliching) [inline connectors/sap/trigger-new-idoc.md]: A simple workaround is to use [Callable recipes](/features/callable-recipes.md). You will create 1 recipe to look for changes to the IDoc, then call other recipes to execute different workflows.

Comment (zweedao) [reply connectors/sap/trigger-new-idoc.md]: the term here is really "listen to IDoc". That's how the trigger works & how SAP people talk.

File: connectors/sap/trigger-new-idoc.md (hunk 1 above)
Comment (zweedao) [reply connectors/sap/trigger-new-idoc.md]: the term here is really "listen to IDoc". That's how this trigger works & how SAP people talk.

I will paraphrase this paragraph a bit.

File: connectors/sap/trigger-new-idoc.md (hunk 2 above)
Comment (zweedao) [reply connectors/sap/trigger-new-idoc.md]: the term here is really "listen to IDoc". That's how this trigger works & how SAP people talk.

**Comment by zweedao** [comment]
@graceliching I've updated to address your review

File: connectors/sap/trigger-new-idoc.md (hunk 3 above)
Comment (graceliching) [reply connectors/sap/trigger-new-idoc.md]: hahahah! oh ok sounds super weird.

**Pull request merged**